api/src/main/java/org/apache/cloudstack/api/response/HostResponse.java

```diff
@@ -302,6 +302,22 @@ public class HostResponse extends BaseResponseWithAnnotations {
     @Param(description = "CPU Arch of the host", since = "4.20")
     private String arch;
 
+    @SerializedName(ApiConstants.STORAGE_ACCESS_GROUPS)
+    @Param(description = "comma-separated list of storage access groups for the host", since = "4.20.1")
+    private String storageAccessGroups;
+
+    @SerializedName("clusterstorageaccessgroups")
+    @Param(description = "comma-separated list of storage access groups on the cluster", since = "4.20.1")
+    private String clusterStorageAccessGroups;
+
+    @SerializedName("podstorageaccessgroups")
+    @Param(description = "comma-separated list of storage access groups on the pod", since = "4.20.1")
+    private String podStorageAccessGroups;
+
+    @SerializedName("zonestorageaccessgroups")
```

Comment:
can add zone/pod/clusterstorageaccessgroups to ApiConstants and use them wherever applicable